values.yaml

```diff
@@ -921,6 +941,8 @@ csi:
     # Extra labels to attach to the vault-csi-provider daemonSet
     # This should be a YAML map of the labels to apply to the csi provider daemonSet
     extraLabels: {}
+    # security context for the pod template in the csi provider daemonSet
+    securityContext: {}
```

Comment:
How come this isn't split between pod and container too?

Reply:
Oh, it is, I just didn't split it out here. I'll go ahead and add those keys.